src/applications/representative-appoint/components/SearchResult.jsx

```diff
@@ -35,23 +41,17 @@ const SearchResult = ({
     (stateCode ? ` ${stateCode}` : '') +
     (zipCode ? ` ${zipCode}` : '');
 
-  // pending analytics event
   const recordContactLinkClick = () => {
-    // recordEvent({
-    //   // prettier-ignore
-    //   'event': 'appoint-a-rep-search-results-click',
-    //   'search-query': query?.locationQueryString,
-    //   'search-filters-list': {
-    //     'representative-type': query?.representativeType,
-    //     'representative-name': query?.representativeQueryString,
-    //   },
-    //   'search-selection': 'Find VA Accredited Rep',
-    //   'search-results-id': representativeId,
-    //   'search-results-total-count':
-    //     searchResults?.meta?.pagination?.totalEntries,
-    //   'search-results-total-pages': searchResults?.meta?.pagination?.totalPages,
-    //   'search-result-position': key,
-    // });
+    // pending analytics event
+  };
+
+  const handleSelect = async selectedRepResult => {
```

Comment:
Does this need to be an async function?

Reply:
Nope :)

https://github.com/department-of-veterans-affairs/vets-website/pull/31857/commits/ac60dc57492233470fd62d4dcf7a7e5f6354d298